src/cljs/org/broadinstitute/firecloud_ui/main.cljs

```diff
@@ -407,6 +433,10 @@
    (fn [{:keys [this state]} token]
      (attempt-auth token (fn [{:keys [success? status-code]}]
                            (cond
+                             (= status-code 502)
+                               (swap! utils/maintenance-mode? not)
```

Comment:
This says, "If you see a 502 and utils/maintenance-mode? is false, make it true, but if it is true, make it false." I don't think that is quite right.